Packs/CiscoUmbrellaReporting/Integrations/CiscoUmbrellaReporting/CiscoUmbrellaReporting.py

```diff
@@ -0,0 +1,1138 @@
+    if "domains" in kwargs:
+        check_valid_indicator_value('domains', kwargs['domains'])
+    page = kwargs.get("page", None)
+    page_size = kwargs.get('page_size', 0)
+    kwargs = pagination(kwargs)
+    params = get_param(kwargs)
+    endpoint = "top-eventtypes"
+    title = get_command_title_string("Event Type", page, page_size)
+    raw_json_response = client.fetch_data_from_cisco_api(endpoint,
+                                                         params).get("data", [])
+    flatten_json_response: List = []
+    if raw_json_response:
+        flatten_json_response = get_flatten_json_response(raw_json_response)
+
+    return CommandResults(
+        readable_output=event_types_lookup_to_markdown(flatten_json_response,
+                                                       title),
+        outputs_prefix=f'{INTEGRATION_CONTEXT_NAME}.EventType',
+        outputs_key_field='',
+        outputs=raw_json_response
+    )
+
+
+def get_activity_list(client: Client, kwargs: Dict):
+    if "ip" in kwargs:
+        check_valid_indicator_value("ip", kwargs['ip'])
+    if "domains" in kwargs:
+        check_valid_indicator_value('domains', kwargs['domains'])
+    page = kwargs.get("page", None)
+    page_size = kwargs.get('page_size', '')
+    kwargs = pagination(kwargs)
+    params = get_param(kwargs)
+    endpoint = "activity"
+    title = get_command_title_string("Activity", page, page_size)
+    raw_json_response = client.fetch_data_from_cisco_api(endpoint,
+                                                         params).get("data", [])
+
+    return CommandResults(
+        readable_output=activity_lookup_to_markdown(raw_json_response, title),
+        outputs_prefix=f'{INTEGRATION_CONTEXT_NAME}.Activity',
+        outputs_key_field='',
+        outputs=raw_json_response,
+    )
+
+
+def get_activity_by_traffic_type(client: Client, kwargs: Dict):
+    traffic_type = kwargs.pop("traffic_type", None)
+    if traffic_type:
+        endpoint = "activity/amp-retrospective" if traffic_type == "amp"\
+            else f"activity/{traffic_type}"
+    else:
+        endpoint = "activity"
+    markdown_function = {
+        "dns": activity_dns_lookup_to_markdown,
+        "proxy": activity_proxy_lookup_to_markdown,
+        "firewall": activity_firewall_lookup_to_markdown,
+        "ip": activity_ip_lookup_to_markdown,
+        "intrusion": activity_intrusion_lookup_to_markdown,
+        "amp": activity_amp_lookup_to_markdown
+    }
+    context_output_name = {
+        "dns": "ActivityDns",
+        "proxy": "ActivityProxy",
+        "firewall": "ActivityFirewall",
+        "intrusion": "ActivityIntrusion",
+        "ip": "ActivityIP",
+        "amp": "ActivityAMPRetro"
+    }
+    traffic_type_params_list = ACTIVITY_TRAFFIC_TYPE_DICT[traffic_type]
+    if not set(kwargs.keys()).issubset(traffic_type_params_list):
+        raise DemistoException(
+            f"Invalid optional parameter is selected for {traffic_type}")
+    if "ip" in kwargs:
```

Comment:
Comments for rows 989 - 1003 are the same as the comments from the `get_destinations_list` function. 
Please update the code in accordance.

Reply:
Done